contracts/shared/AirdropAdmin.sol

```diff
@@ -13,6 +13,10 @@ abstract contract AirdropAdmin {
 
     event AdminChanged(address indexed adminAddress);
 
+    constructor(address _admin) {
+        admin = _admin;
```

Comment:
It is not a bad idea to emit `AirdropChanged` event here. For example if you want to have that data tracked in subgraph, you can just look for the events. Without it you'd need to query blockchain